aws/adapter.go

```diff
@@ -124,6 +125,10 @@ const (
 	DefaultAlbS3LogsPrefix = ""
 
 	DefaultCustomFilter = ""
+
+	// DefaultZoneAffinity specifies dns_record.client_routing_policy, see also https://docs.aws.amazon.com/elasticloadbalancing/latest/network/network-load-balancers.html#load-balancer-attributes
+	DefaultZoneAffinity = "any_availability_zone"
```

Comment:
Maybe highlight that it is safe to change as per https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-elasticloadbalancingv2-loadbalancer-loadbalancerattribute.html#aws-properties-elasticloadbalancingv2-loadbalancer-loadbalancerattribute-properties

Reply:
done